packages/astro/src/transitions/router.ts

```diff
@@ -372,7 +383,21 @@ async function transition(
 	}
 }
 
+let navigateOnSeverWarned = false;
+
 export function navigate(href: string, options?: Options) {
+	
+	if (!supportsViewTransitions) {
```

Comment:
For treeshaking, I made some tests and I think this condition needs to be `if (import.meta.env.SSR)` for it to work, otherwise the client would include the warning. Since `document.startViewTransition` is non-static.

Reply:
Good catch!

Reply:
There's still the typo in case you missed it 👀

Reply:
thanks, corrected